src/main/java/org/broadinstitute/hellbender/engine/VariantWalker.java

```diff
@@ -63,6 +63,54 @@ protected void initializeDrivingVariants() {
         //Note: the intervals for the driving variants are set in onStartup
     }
 
+    /**
+     * A method to allow a user to inject data sources after initialization that were not specified as command-line
+     * arguments.
+     * @return The {@link FeatureInput} used as the key for this data source.
+     */
+    @SuppressWarnings({"unchecked", "rawtypes"})
+    protected FeatureInput<? extends Feature> injectFeatureDataSourcesAfterInitialization(final String filePath,
```

Comment:
If you move this method up to `GATKTool`, you won't have to expose `GATKTool.features` (can keep it package-private)

Reply:
Also, a better name would be `addFeatureInputsAfterInitialization()`

Reply:
Sounds good.  Fixed!